bedrock/pocket/templates/pocket/pocket-and-firefox.html

```diff
@@ -25,21 +25,21 @@
   media_class='mzp-l-split-h-center',
   image_url='img/pocket/save-inspirations.png',
   )%}
-  <span class="tagline">Pocket for Firefox</span>
-  <h1 class="section-heading">Build your personal library of fascinating reads.</h1>
-  <p class="section-lede">Included inside Firefox, the <span class="pocket-logo-inline">Pocket Logo</span> button lets you save articles from across the web and read them in a quiet, private space.</p>
+  <span class="tagline">{{ ftl('pocket-features-pocket-and-firefox') }}</span>
+  <h1 class="section-heading">{{ ftl('pocket-features-build-your-personal') }}</h1>
+  <p class="section-lede">{{ ftl('pocket-features-included-inside') }}</p>
   <div class="section-ctas">
     {{ pocket_fxa_button(entrypoint='pocket', class_name='new-tab-pocket-cta', button_text='<span class="mzp-c-logo mzp-t-logo-sm mzp-t-product-family"></span>Activate Pocket in Firefox', optional_parameters={'s': 'fflearnmore', 'utm_campaign': 'landing-page', 'utm_content': 'page-button'}, optional_attributes={'data-cta-text': 'Activate Pocket', 'data-cta-type': 'activate pocket', 'data-cta-position': 'primary'}) }}
```

Comment:
I'm having a lil' challenging time adding a `ftl` string here.
I thought this would work: 
```suggestion
    {{ pocket_fxa_button(entrypoint='pocket', class_name='new-tab-pocket-cta', button_text='<span class="mzp-c-logo mzp-t-logo-sm mzp-t-product-family"></span>ftl('pocket-features-activate-pocket)'', optional_parameters={'s': 'fflearnmore', 'utm_campaign': 'landing-page', 'utm_content': 'page-button'}, optional_attributes={'data-cta-text': 'Activate Pocket', 'data-cta-type': 'activate pocket', 'data-cta-position': 'primary'}) }}
```
and tried other syntax variations of it, however still getting syntax errors on the page.
I tried copying what I [saw in other templates ](https://github.com/mozilla/bedrock/blob/main/bedrock/firefox/templates/firefox/accounts.html#L80), and removing the quotes (just to test things around), and yet I am still having trouble adding a fluent string.

Any suggestions?

Reply:
This seems to work ok:

```suggestion
    {{ pocket_fxa_button(entrypoint='pocket', class_name='new-tab-pocket-cta', button_text='<span class="mzp-c-logo mzp-t-logo-sm mzp-t-product-family"></span>'|safe + ftl('pocket-features-activate-pocket'), optional_parameters={'s': 'fflearnmore', 'utm_campaign': 'landing-page', 'utm_content': 'page-button'}, optional_attributes={'data-cta-text': 'Activate Pocket', 'data-cta-type': 'activate pocket', 'data-cta-position': 'primary'}) }}
```

Reply:
thank you!!